src/commands/cmd_constraint.c

```diff
@@ -403,6 +406,10 @@ int Graph_Constraint
 	const char *props_cstr[prop_count];
 	for(uint8_t i = 0; i < prop_count; i++) {
 		props_cstr[i] = RedisModule_StringPtrLen(props[i], NULL);
+		if(str_MatchRegex("[a-zA-Z_][a-zA-Z0-9_$]*", props_cstr[i], 0, strlen(props_cstr[i])) == false) {
+			RedisModule_ReplyWithError(ctx, "At least one property name is invalid");
```

Comment:
What do you think of using a message that helps the user to detect the invalid property name?, may be:
``Property name at position %i is invalid: props_cstr[i]``

Reply:
Good idea, but "position %i" is ambiguous: is it the i'th argument or the i'th property name? Is this 0-based on 1-based?
If you want to go the extra mile: "The x property name is invalid: " + props_cstr[i-1], where x is "1st", "2nd", "3rd", ... (I guess you'll want a utility function for convert i to str).

Reply:
@LiorKogan So we are marking this as resolved?

Reply:
The current error message is fine, but @nafraf's suggestion is better. Up to you.

Reply:
@nafraf the reason why I didn't do it is because in non of the modules we use format in the error replies.
I think I'll add a new infra for this.